charts/azuredisk-csi-driver/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.1.0-alpha"
```

Comment:
let's use `v0.2.0-alpha`

Reply:
OK.
keep `appVersion` = `image version`?

Reply:
` appVersion` here app means application, right? then it should be image version.

Reply:
Yes